tests/bin/unix_echo.rs

```diff
@@ -0,0 +1,14 @@
+use std::io::{self, Read, Write};
+use std::os::unix::net::{UnixListener, UnixStream};
+
+fn main() -> io::Result<()> {
+    let listener = UnixListener::bind("/tmp/enarx_unix_echo_test_to_test")?;
```

Comment:
I might call this something like `/tmp/enarx_unix_echo_to_bin` and `/tmp/enarx_unix_echo_from_bin` (or `echo_from_code` or whatever sounds good) to be a bit more descriptive.

Reply:
thanks! good idea